wire/msgtx.go

```diff
@@ -106,24 +106,26 @@ const (
 	// an input's witness data. This number is derived from the fact that
 	// for script validation, each pushed item onto the stack must be less
 	// than 10k bytes.
-	maxWitnessItemSize = 11000
+	maxWitnessItemSize = 4_000_000
```

Comment:
why `4_000_000` instead of `4000000`?

Reply:
Readability!?

Reply:
@otech47 thx for taking a look! It's the same thing, just using a literal to demarcate things in place of a comma (not valid syntax in Go).

Reply:
https://go.dev/doc/go1.13

Reply:
Why not 400_000?

That seems to be the maximum weight for transactions after Segwit support:
https://github.com/bitcoin/bitcoin/blob/master/src/policy/policy.h#L27

Reply:
The reason we don't use 400_000 is, bc thats only the policy rule of bitcoin-core nodes not the consensus of the bitcoin network. So transactions > 400_000 are considered non-standard but do comply with the consensus rules. As long as they are not bigger than the max_block size thats why we use 4_000_000

https://bitcoin.stackexchange.com/questions/100317/what-is-the-difference-between-policy-and-consensus-when-it-comes-to-a-bitcoin-c

Reply:
Thank for your answer.